pkgs/development/compilers/graalvm/community-edition.nix

```diff
@@ -1,48 +1,60 @@
-{ stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath }:
+{ stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath, lib, darwin, openssl }:
 
 let
+  platform = if stdenv.isDarwin then "darwin-amd64" else "linux-amd64";
```

Comment:
can we just have these use `stdenv.system` rather than introducing new platform identifiers?

Reply:
Unfortunately not, because the `graalvm` release artifacts use "linux-amd64" in the url rather than "x86_64-linux" (= `stdenv.system`).

Reply:
sure, but we could map to that with an attribute set, right?

Reply:
I'm sorry, but I don't get what that means. Could you give/point me to an example?

Reply:
actually upon looking further I agree that this probably isn't worth it